pkg/kubectl/run.go

```diff
@@ -193,27 +190,9 @@ func getEnvs(genericParams map[string]interface{}) ([]api.EnvVar, error) {
 	return envs, nil
 }
 
-func getV1Envs(genericParams map[string]interface{}) ([]v1.EnvVar, error) {
```

Comment:
Back when Job internals were in extensions, and I was adding generator for `batch/v1` I had to create a Job targeted against that endpoint, which resulted in having copies of several methods in `run.go`. With this PR I'm removing those duplicates.